packages/libxlsxwriter/build.sh

```diff
@@ -17,11 +17,11 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 termux_step_post_get_source() {
 	# Do not forget to bump revision of reverse dependencies and rebuild them
 	# after SOVERSION is changed.
-	local _SOVERSION=7
+	local _SOVERSION=8
 
 	local v=$(sed -En 's/.*LXW_SOVERSION .*"(.*)".*/\1/p' \
 			include/xlsxwriter.h)
 	if [ "${v}" != "${_SOVERSION}" ]; then
-		termux_error_exit "SOVERSION guard check failed."
+		termux_error_exit "SOVERSION guard check failed. expected ${v}"
```

Comment:
Probably it should be `Expected ${_SOVERSION}, got ${v}.`

Reply:
Thanks. I have changed that.

Reply:
Probably we should change it tree-wide, for all packages with soversion guards. With `[no ci]` of course.